vars/sectionNightlyTests.groovy

```diff
@@ -38,6 +38,14 @@ def call(PipelineCallbacksRunner pcr, AppPipelineConfig config, PipelineType pip
       }
     }
 
+    fortifyScan(
+      pipelineCallbacksRunner: pcr,
+      appPipelineConfig: config,
+      builder: builder,
+      environment: environment.nonProdName,
```

Comment:
Agree with Neil's previous comment, even i think we should get rid of `environment` from here even if it means we should make setting keyvault mandatory for everyone. This makes it look that his step is environment specific while it's not.

Reply:
The implementation is parameterised to accept a different environment later on. Could that be treated a non-blocker, minor comment?

Reply:
sure, if others are happy with it.

Reply:
does this need a team specific key vault? or can it just use a central username and password?

Reply:
It is indeed team specific.

Reply:
We should remove references to an `environment` - the check doesn't depend on _any_ environment. It's not a question of allowing teams to specify a different environment - including the `environment` as a parameter may make people think it's environment specific.

Reply:
You could pull up passing the vault name to here, which means you don't need to force everyone to set the vault name and can still rely on convention.

That may mean you need to pull up the `if enabled` check to around this function

Reply:
Environment variable has been removed and we now completely rely on the vaultValue which is passed through enableFortifyScan(). Can you please have a look? By the way, some tests are failing in the pipeline, but not in my local.

https://sandbox-build.platform.hmcts.net/blue/organizations/jenkins/HMCTS_Sandbox_Pipeline_Test%2Fcnp-jenkins-library/detail/PR-755/17/tests

Reply:
is there a reason you didn't pass the vault name down from here allowing a convention based approach like suggested above?

Reply:
@timja, Kaan and I have got a little confused amongst various comments. Could you please suggest a commit or just push one to the branch? Could you please also resolve other comments you were satisfied with the actions/responses for?

Reply:
@timja I have tried to follow the same login which has been applied to enableDbMigration('ccd'). But I don't understand if we don't force people to pass it, what we would put for the vault name. 
I guess you want me to make below change. So, how should we handle vault name if users don't pass it

    if (config.fortifyScan) {
      fortifyScan(
        pipelineCallbacksRunner: pcr,
        fortifyVaultName: config.fortifyVaultName,
        builder: builder,
        product: product,
      )
    }

Reply:
something like:

```
if (config.fortifyScan) {
  fortifyScan(
    pipelineCallbacksRunner: pcr,
    fortifyVaultName: config.fortifyVaultName ?: "https://${var.product}-${environment.nonProdName}.vault.azure.net/",
    builder: builder,
    product: product,
  )
}
```

Reply:
We can do that. What I understand from @njrich28 comment [here](https://github.com/hmcts/cnp-jenkins-library/pull/755#discussion_r665229566), we shouldn't use environment name anywhere in the vault url, as this make people confused. Now, I am making the above change.